### ISSUE TITLE
Link to npm package for zen-observable

### ISSUE THREAD

**Issue by SharpEdgeMarshall** [apollographql/apollo-link#14]
zen-observable dependency is causing issues during install with docker node image because needs git to download.

Please change the git form
```
"zen-observable": "git+https://github.com/evanshauser/zen-observable.git"
```
to npm form
```
"zen-observable": "^0.5.2"
```

**Comment by jbaxleyiii** [comment]
@evanshauser what changes did you make to zen that required using your fork? I agree that we should try and use npm if possible

**Comment by evanshauser** [comment]
@SharpEdgeMarshall thank you for pointing the issue out!

@jbaxleyiii I changed how the modules were exported: https://github.com/zenparsing/zen-observable/issues/31. Thoughts on better ways to get around the issue?

**Comment by Frizi** [comment]
Having the same issue. As a workaround, I'm temporarily changing that manually in my `yarn.lock` to point to npm version and it appears to work in my project.

```
apollo-link@^0.0.3:
  version "0.0.3"
  resolved "https://registry.yarnpkg.com/apollo-link/-/apollo-link-0.0.3.tgz#9b02db8def792688e14f8be8b30f716f7789acb9"
  dependencies:
    apollo-fetch "^0.3.0"
    graphql "^0.10.3"
    graphql-tag "^2.4.2"
    zen-observable "^0.5.2"
  optionalDependencies:
    "@types/graphql" "^0.9.4"
    "@types/zen-observable" "^0.5.1"

zen-observable@^0.5.2:
  version "0.5.2"
  resolved "https://registry.yarnpkg.com/zen-observable/-/zen-observable-0.5.2.tgz#845a1cfda804a13419ec8ec12343e1f4c45cd07a"
```

**Comment by katopz** [comment]
Aw! I'm lost 2 hours just because of this, I'll paste log here for google crawling.
```shell
npm http request GET https://registry.npmjs.org/apollo-fetch
npm info attempt registry request try #1 at 09:11:36
npm http request GET https://registry.npmjs.org/@types%2fzen-observable
npm ERR! git clone --template=/root/.npm/_git-remotes/_templates --mirror https://github.com/evanshauser/zen-observable.git /root/.npm/_git-remotes/git-https-github-com-evanshauser-zen-observable-git-5d016840: undefined
npm ERR! git clone --template=/root/.npm/_git-remotes/_templates --mirror https://github.com/evanshauser/zen-observable.git /root/.npm/_git-remotes/git-https-github-com-evanshauser-zen-observable-git-5d016840: undefined
npm http 200 https://registry.npmjs.org/apollo-fetch
npm info retry fetch attempt 1 at 09:11:37
npm info attempt registry request try #1 at 09:11:37
npm http fetch GET https://registry.npmjs.org/apollo-fetch/-/apollo-fetch-0.3.0.tgz
npm http fetch 200 https://registry.npmjs.org/apollo-fetch/-/apollo-fetch-0.3.0.tgz
npm http 200 https://registry.npmjs.org/@types%2fzen-observable
npm info addNameTag [ '@types/zen-observable', 'latest' ]
npm info retry fetch attempt 1 at 09:11:38
npm info attempt registry request try #1 at 09:11:38
npm http fetch GET https://registry.npmjs.org/@types/zen-observable/-/zen-observable-0.5.2.tgz
npm http fetch 200 https://registry.npmjs.org/@types/zen-observable/-/zen-observable-0.5.2.tgz
npm ERR! code ENOGIT
npm ERR! not found: git
```
Pretty please fix this, I just don't want to add workaround.

Thanks

**Comment by evanshauser** [comment]
@SharpEdgeMarshall @Frizi @katopz I exported a new module for zen-observable that should mean the installation won't need `git`. Please see [this commit](https://github.com/apollographql/apollo-link/commit/15a2f52fe7a520a78283cd86e2f4a9c2207de717). Thank you for sharing your issues. Let me know if you continue facing problems

Using `zen-observable-exported` is a temporary solution, pending resolution on: 
 * https://github.com/zenparsing/zen-observable/pull/33
 * https://github.com/zenparsing/zen-observable/issues/31

**Comment by evanshauser** [comment]
Fixed with #23

**Comment by evanshauser** [comment]
Please open another issue if something like this crops up again. Thank you @SharpEdgeMarshall @Frizi @katopz  for bringing it to my attention
